src/main/java/com/chat/yourway/service/ContactService.java

```diff
@@ -0,0 +1,24 @@
+package com.chat.yourway.service;
+
+import com.chat.yourway.dto.request.ChangePassword;
+import com.chat.yourway.repository.ContactRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class ContactService {
```

Comment:
Could you find and install "google code style schema" (for example) for IDEA? It would help you with auto-formatting and save your time